.cargo/config

```diff
@@ -23,5 +23,29 @@ ruma-clippy = """\
     -W clippy::wildcard_imports \
 """
 
+exhaustive-clippy = """\
+  clippy --workspace --all-targets --features=full,compat,unstable-pre-spec -- \
```

Comment:
Can you just use this feature set in `ruma-clippy`? I'll adjust features a bit afterwards (to also activate the ruma-client ones w/o making this invocation super long).